### PR TITLE
module 1: Add Contact branch on the Mobile Version

### PR DESCRIPTION
**Pull request by amare1990**
- Add contact form at the bottom of the project
- Add valid required fields for the input fields
- Add the type of the button in the contact form to be submit type
- Add meaningful text in the placeholder text.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -170,10 +170,11 @@ <h2 class="head">contact me</h2>
         <p class="paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
           I’d love to help with it</p>
              <form action="https://formspree.io/f/xknegber" method="post">
-                <input type="text" placeholder="Alexendra loremipsum" class="name" minlength="30" maxlength="30" required>
-                <input type="email" placeholder="alexendra@gmail.com" class="em" required>
+                <input type="text" placeholder="Enter your full name" class="name" minlength="30" maxlength="30" required>
```
Comment (aimemalaika) [inline index.html]: - [ ] Kindly change the minimum number of characters for the name input as it is not allowing me to submit if the names has less then 30 characters

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @azizunnabi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @amare1990 and @azizunnabi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used

## Optional suggestions

- [ ] [optional] Kindly ensure your design is flexible currently it seems you have a fixed width in your design, also the header background seems misaligned kindly look at it.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by azizunnabi** [comment]
After passing all Linters errors and and approval i am going to merge it nwo

**Pull request merged**